target/linux/ath79/image/generic.mk

```diff
@@ -630,15 +687,18 @@ TARGET_DEVICES += netgear_wndr3700v2
 define Device/pisen_wmm003n
   $(Device/tplink-8mlzma)
   ATH_SOC := ar9331
-  DEVICE_TITLE := Pisen WMM003N (Cloud Easy Power)
+  DEVICE_VENDOR := PISEN
+  DEVICE_MODEL := WMM003N
+  DEVICE_VARIANT := Cloud Easy Power
```

Comment:
From very quick research, it looks like WMM003N and "Cloud Easy Power" are alternative names for the same device, but would not really match DEVICE_VARIANT. I suggest keeping both in DEVICE_MODEL, so one of
`DEVICE_MODEL := WMM003N (Cloud Easy Power)`
`DEVICE_MODEL := Cloud Easy Power(WMM003N)`
where I actually favor the latter, as "Cloud Easy Power" seems to be the marketed name.